packages/core/src/Popover.js

```diff
@@ -49,12 +63,15 @@ function Popover({
 }
 
 Popover.propTypes = {
+    onClick: PropTypes.func,
     placement: anchoredPropTypes.placement,
     arrowStyle: anchoredPropTypes.arrowStyle,
     nodeRef: anchoredPropTypes.nodeRef,
+    handleInsideClick: PropTypes.func.isRequired,
```

Comment:
Maybe call it `onInsideClick` since it's a handler prop for component user?
```suggestion
    onInsideClick: PropTypes.func.isRequired,
```

Reply:
Yeah sounds good to me. Thanks for the tip